dashboard/test/test_helper.rb

```diff
@@ -333,3 +333,7 @@ def write(json)
     @records << json
   end
 end
+
+def json_response
+  ActiveSupport::JSON.decode @response.body
```

Comment:
I don't really remember, but I doubt I had a good reason. `JSON.parse` is much better 👍